app/src/androidTest/java/com/sdp/movemeet/view/profile/DrawableMatcher.java

```diff
@@ -0,0 +1,68 @@
+package com.sdp.movemeet.view.profile;
```

Comment:
So if I understand correctly, this is a utility class only used for comparing the images in the newly added tests, right ?

Reply:
That's it Victor! And this class is actually used only for the test `EditProfileActivityImagePickerTest`. It actually implements an Espresso Matcher for ImageView and I found it [here](https://medium.com/@dbottillo/android-ui-test-espresso-matcher-for-imageview-1a28c832626f). I will document it today!